src/main/java/io/electrum/prepaidutility/api/TokenPurchasesResource.java

```diff
@@ -83,6 +83,17 @@
       }
    }
 
+   public class TrialTokenPurchaseRequest {
+      public static final String TRIAL_TOKEN_PURCHASE_REQUEST = "trialTokenPurchaseRequest";
+      public static final int SUCCESS = 200;
+      public static final String RELATIVE_PATH = "/{" + PathParameters.PURCHASE_ID + "}/trials";

```

Comment:
Including the _purchase_ ID in the path like this might lead people to send many trial purchases with the same ID and then, also, an actual purchase with the same ID again. This wouldn't work for us since we'd identify each subsequent usage of the ID again as a duplicate request. Perhaps the following URL might help avoid that issue (or at least not steer people towards it):
```suggestion
      public static final String RELATIVE_PATH = "/trials/{" + PathParameters.TRIAL_ID;
```